application/controllers/admin/Quotas.php

```diff
@@ -271,13 +304,18 @@ public function index(int $iSurveyId, bool $quickreport = false)
      */
     public function insertquotaanswer(int $iSurveyId)
     {
+        $this->requirePostRequest();
+
         $iSurveyId = sanitize_int($iSurveyId);
+        $qid = Yii::app()->request->getPost('quota_qid');
+        $quota_id = Yii::app()->request->getPost('quota_id');
         $this->checkPermissions($iSurveyId, 'update');
+        $this->checkValidQuotaId($surveyId, $quota_id);
```

Comment:
@Shnoulle please be a bit more careful when changing variable names, $surveyId should be $iSurveyId here. Code assistance in IDE marks this directly as red error, since this variable is not set. There was another issue in ls 6 with the same problem.